DownloadManager/Plugin.DownloadManager.Android/Plugin.DownloadManager.Android.csproj

```diff
@@ -16,8 +16,9 @@
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
     <DevInstrumentationEnabled>True</DevInstrumentationEnabled>
-    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>
-    <AndroidTlsProvider></AndroidTlsProvider>
+    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>

```

Comment:
Will this not require everyone, using this package, to upgrade his app to at least this version? I'm sometimes quite concerned to do this, not to lock people in a position where they're forced to update if they want to continue using my plugin.

I haven't been involved in the Xamarin development lately and don't know their best practices. Please write your comment here.

Reply:
I dunno best practices for this, but I have checked the docs: https://docs.microsoft.com/en-us/xamarin/android/app-fundamentals/android-api-levels?tabs=windows

> We recommend that you always compile with the latest available Target Framework version. Doing so provides you with helpful warning messages for any deprecated APIs that might be called by your code. Using the latest Target Framework version is especially important when you use the latest support library releases – each library expects your app to be compiled at that support library's minimum API level or greater.

What API calls you make, determinate how low you can go on the "Minimum android version", but it still compiles at eg 9.0.

I have _some_ knowledge about this, but not at all an expert 👍